src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
That's really surprising.

Because in this page http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html, it shows the transformation matrix for Srgb D50 which exactly matches the http://ninedegreesbelow.com/photography/srgb-color-space-to-profile.html#sRGB-profile-source-destination-white-points  table7 and also the icc profile values. Theae are calculated through chromatic adaptation of the Srgb primaries. 

I wonder if the two methods give the same result.